lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -454,7 +464,7 @@ public function add(string $sqlPartName, $sqlPart, bool $append = false) : self
      *         ->leftJoin('u', 'phonenumbers', 'p', 'u.id = p.user_id');
      * </code>
      *
-     * @param mixed $select The selection expressions.
+     * @param string|string[] $select The selection expressions.
```

Comment:
This will probably change after a rebase. Please rebase first.

Reply:
Indeed, lots of changes upstream! Rebased now.
I love the variadics for the expressions, this is precisely what I was about to tackle next.